### PR TITLE
Implementation of CRUD (Create , delete , update and delete)

### PR DESCRIPTION
**Pull request by Yorsyboy**
In this pull request i:

- Implemented a function for adding a new task (add a new element to the array).
- Implemented a function for deleting a task (remove all elements from the array).
- Implemented a function for editing a task (remove all elements from the array).
- All changes to the To-Do List are saved in local storage
- implemented the interactivity to the list

### CODE REVIEW

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Yorsyboy , 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-04-12 20-44-47

# To Highlights

-  Pull Request has a great title and summary :heavy_check_mark: 
- The README file is well structured and descriptive :+1: 
- Linters checks are all passing with green check marks :heavy_check_mark: 


## Required Changes ♻️

- [ ] Most of the functionalities in the ` index.js` file should have been added in a new JS file. This is what the requirement of this project stipulate. Kindly create a new JS file for these amazing functionalities in this file.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -1,33 +1,151 @@
...
+    });
+
+  document
+    .getElementById('clear-completed-button')
+    .addEventListener('click', () => {
+      clearLocalStorage();
+    });
+});
+
+document.addEventListener('load', updateView());
```
Comment (FrederickMih) [inline src/index.js]: - [ ] Hi @Yorsyboy, your app is really fantastic:+1: 

     Most of the functionalities in this file should have been added in a new JS file. This is what the requirement of this project stipulate. Kindly create a new JS file for these amazing functionalities in this file.

**Comment by Yorsyboy** [comment]
@FrederickMih can't find any comments under review

**Comment by FrederickMih** [comment]
> @FrederickMih can't find any comments under review

Hi @Yorsyboy, Please can you refresh once more?. 

I added an inline comment and forgot to submit it. Sorry, about that.

**Comment by Yorsyboy** [comment]
@FrederickMih I have refreshed multi times , still not getting a review

**Comment by FrederickMih** [comment]
> @FrederickMih I have refreshed multi times , still not getting a review

@Yorsyboy , I just update the comment once more. Please check once more. If still, you can't see any comments, I will reach out to you on slack.

**Comment by FrederickMih** [comment]
> > @FrederickMih I have refreshed multi times , still not getting a review
> 
> @Yorsyboy , please below :point_down:  is the comment I dropped. Kindly adjust the changes and submit your project once more for approval.

- [ ]  Most of the functionalities in index.js file should have been added in a new JS file. This is what the requirement of this project stipulate. Kindly create a new JS file for these amazing functionalities in this file.

   thanks :pray:

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Yorsyboy , 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-04-12 20-44-47

# To Highlights

-  Pull Request has a great title and summary :heavy_check_mark: 
- The README file is well structured and descriptive :+1: 
- Linters checks are all passing with green check marks :heavy_check_mark: 


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
# Status: Approved :tada: :tada: 



Hi @Yorsyboy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**